.github/actions/setup-test-environment/action.yml

```
@@ -57,7 +57,7 @@ runs:
         echo "Building Llama Stack"
 
         LLAMA_STACK_DIR=. \
-          uv run --no-sync llama stack build --template ci-tests --image-type venv
+          uv run --no-sync llama stack list-deps ci-tests --format deps-only | xargs -L1 uv pip install
```

can drop now that deps-only is the default 
```suggestion
          uv run --no-sync llama stack list-deps ci-tests | xargs -L1 uv pip install
```